packages/core/src/spec/lib/jdk/KeyToolSpec.ts

```diff
@@ -64,7 +66,8 @@ describe('KeyTool', () => {
     } as CreateKeyOptions;
 
     it('Executes the correct command to create a key', async () => {
-      const keyTool = new KeyTool(jdkHelper);
+      const mockLog = new MockLog();
+      const keyTool = new KeyTool(jdkHelper, mockLog);
```

Comment:
Since we're not doing anything with MockLog (such as checking that certain things are output), we can just do `new KeyTool(jdkHelper, new MockLog())`.